go.mod

```diff
@@ -103,137 +101,171 @@ require (
 )
 
 require (
-	cloud.google.com/go v0.81.0 // indirect
+	github.com/cilium/cilium v1.11.0-rc0
```

Comment:
This is a release candidate, can we use a stable version of the package?

Reply:
Ouch... Thanks for finding that. I missed it totally.

Reply:
Found it. 

`github.com/envoyproxy/go-control-plane v0.9.10-0.20210907150352-cf90f659a021`

This breaks `pkg/controller/nodeport-proxy/envoymanager`. 
If I downgrade it to 9.9 (with which envoymanager works), cilium package gets downgraded to rc0 above. 

I guess I have to fix `pkg/controller/nodeport-proxy/envoymanager` to work with `go-control-plane v0.9.10-0`.